Symfony/CS/Tests/Fixer/Symfony/MultilineArrayTrailingCommaFixerTest.php

```diff
@@ -298,7 +260,78 @@ function foo(array $a)
         "string"
         /* foo */);',
             ),
-            array(
+        );
+    }
+
+    /**
+     * @dataProvider provideExamples54
+     * @requires PHP 5.4
+     */
+    public function testFix54($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideExamples54()
+    {
+        return array(
+                // short syntax tests
+                array('<?php $x = array([]);'),
+                array('<?php $x = [[]];'),
```

Comment:
funny thing that we actually _can_ handle short array even under PHP 5.4. And this is not the only fixer that do it perfectly.